### PR TITLE
LPD-23801 Show new error messages when a user tries to publish an article that an admin is currently editing

### PR DESCRIPTION
**Pull request by marcogalluzzi**
To test, follow the steps in https://liferay.atlassian.net/browse/LPD-23801

### CODE REVIEW

File: modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties [hunk 1]
```diff
@@ -19285,6 +19285,7 @@ this-area-is-defined-by-the-theme.-you-can-change-the-theme-settings-by-clicking
 this-article-cannot-be-published-because-its-parent-has-not-been-published=This article cannot be published because its parent has not been published yet.
 this-article-is-configured-to-use-a-display-page-that-does-not-exist-on-the-current-site=This article is configured to use a display page that does not exist on the current site. For it to function properly, the page with UUID {0} must be imported, or a new display page must be selected.
 this-article-is-currently-being-edited-by-another-user=This article is currently being edited by another user. Actions like edit, expire, move, and delete are temporarily unavailable.
+this-article-is-now-under-control-of-x=This article is now under control of {0}. Please note that the last changes can not be saved.
```
Comment (adolfopa) [inline modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties:19288]: @austinchiang Can you double check this literal? It sounds weird to me.

**Review by adolfopa — CHANGES_REQUESTED** [review]
The code looks fine; I've asked help from Austin to review the key.

File: modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties (hunk 1 above)
Comment (marcogalluzzi) [reply modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties]: Thanks @austinchiang . I've done the change @adolfopa

**Comment by marcogalluzzi** [comment]
ci:forward

**Review by ambrinchaudhary — APPROVED** [review]

**Comment by marcogalluzzi** [comment]
ci:test:stable

**Comment by marcogalluzzi** [comment]
Closing this pull request in favor of #5404 to avoid this [CI error](https://liferay.atlassian.net/browse/LRCI-4215)

**Pull request closed without merging**